### PR TITLE
damian

### PR DESCRIPTION
**Pull request by SonixApache**

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
damian

**Comment by Kharkov72** [comment]
this is definitely me when I'm damian

**Review by DamianX — APPROVED** [review]
damian

File: html/changelogs/damian.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+author: DamianQ
```
Comment (Kurfursten) [inline html/changelogs/damian.yml:1]: 👌

**Review by Kurfursten — APPROVED** [review]

**Comment by ShiftyRail** [comment]
What would a "Damian" antag look like ?

File: html/changelogs/damian.yml (hunk 1 above)
Comment (ShiftyRail) [reply html/changelogs/damian.yml:1]: Technically would have to be DamianX here.

**Review by ghost — APPROVED** [review]
I can say with confidence that damianposting is definitely not getting out of hand

**Comment by Irradiation** [comment]
Damian

**Pull request closed without merging**